tests/hap/test_run_svmpoller.py

```diff
@@ -60,29 +54,34 @@ def test_run_svmpoller(tmpdir, dataset):
         Success Criteria:
             The SVM processing returns a value of 0: Success or 1: Failure
 
-        The input master_list file is a list of poller filenames, one filename per line.
+        The input svm_list file is a list of poller filenames, one filename per line.
         Each poller file must be obtained from a specified directory and read to obtain the
         names of the data files which need to be processed.
 
         This test file can be executed in the following manner:
             $ pytest -n # -s --basetemp=/internal/hladata/yourUniqueDirectoryHere --bigdata --slow
-              --master_list /internal/hladata/input/master_poller_list.txt test_run_svmpoller.py >&
-              test_svmpoller_output.txt &
+              --svm_list svm_input.lst test_run_svmpoller.py >& test_svmpoller_output.txt &
             $ tail -f test_svmpoller_output.txt
           * The `-n #` option can be used to run tests in parallel if `pytest-xdist` has
             been installed where `#` is the number of cpus to use. THIS IS NOT ADVISED FOR USE.
           * Note: When running this test, the `--basetemp` directory should be set to a unique
             existing directory to avoid deleting previous test output.
-          * A default master list exists in the tests/hla directory and contains 121 datasets.  This
-            is probably NOT the list you want to use, but it allows you to see what this file should
-            contain.
+          * A default master list, svm_input.lst, exists in the tests/hla directory and contains 3 datasets.
+            This specific list may NOT the list you want to use, but it allows you to see what this file
+            should contain.  Please note the PyTests should be kept to runtimes which are not
+            excessive.
 
     """
     print("TEST_RUN_SVMPOLLER. Dataset: ", dataset)
 
     current_dt = datetime.datetime.now()
     print(str(current_dt))
 
+    # Create working directory specified for the test
+    #install_dir = os.path.dirname(__file__)
+    #local_path = os.path.join(install_dir, dataset)
+    #os.chdir(install_dir)
+
```

Comment:
Code that has been commented out can be simply deleted if it is not needed for the final working version.

Reply:
Fixed in e29c47d.